src/commands/mirror_node.mjs

```diff
@@ -158,7 +159,9 @@ export class MirrorNodeCommand extends BaseCommand {
             flags.mirrorNodeVersion
           ])
 
-          await prompts.execute(task, self.configManager, MirrorNodeCommand.DEPLOY_FLAGS_LIST)
+          if (!MirrorNodeCommand.DEPLOY_FLAGS_LIST.includes(flags.quiet)) {
```

Comment:
instead of doing the checks here, if we do it here:
https://github.com/hashgraph/solo/blob/ea01a94088ec09eb7848441b947c5e7377f2219a/src/commands/prompts.mjs#L490-L490
```
  if (configManager.getFlag(flags.quiet)) {
    return
  }
```
then we can remove all of these if checks.